shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -662,7 +662,8 @@ public void detachFromFlutterEngine() {
             + " connection to the engine "
             + getFlutterEngine()
             + " evicted by another attaching activity");
-    release();
+    delegate.onDestroyView();
```

Comment:
please check for `delegate != null` before calling.

Reply:
Also, since you are making this change, could `release` also perform that check?